mkosi/__init__.py

```diff
@@ -830,6 +830,47 @@ def run_finalize_scripts(context: Context) -> None:
                 )
 
 
+def run_postoutput_scripts(context: Context) -> None:
+    if not context.config.postoutput_scripts:
+        return
+
+    env = dict(
+        DISTRIBUTION=str(context.config.distribution),
+        RELEASE=context.config.release,
+        ARCHITECTURE=str(context.config.architecture),
+        SRCDIR="/work/src",
+        OUTPUTDIR="/work/out",
+        MKOSI_UID=str(INVOKING_USER.uid),
+        MKOSI_GID=str(INVOKING_USER.gid),
+        MKOSI_CONFIG="/work/config.json",
```

Comment:
These three are still missing in the environment variable table.

Reply:
Dunno how I mist those